bundles/framework/publisher2/view/PanelLayout.js

```diff
@@ -161,10 +119,10 @@ Oskari.clazz.define('Oskari.mapframework.bundle.publisher2.view.PanelLayout',
             // the values stored under mapfull's conf.
             me.values = {
                 metadata: {
-                    style: {
-                        font: jQuery('select[name=publisher-fonts]').val() ? jQuery('select[name=publisher-fonts]').val() : null
-                    },
-                    theme: Oskari.app.getTheming().getTheme()
+                    theme: Oskari.app.getTheming().getTheme(),
+                    styles: {
```

Comment:
This was changed from `style` to `styles` but other code changes indicates that it wasn't on purpose/other code assumes it's still under `style`